portal-kernel/src/com/liferay/portal/kernel/model/version/VersionedModel.java

```diff
@@ -23,7 +23,7 @@
 
 	public long getPrimaryKey();
 
-	public boolean isDraft();
+	public boolean isHead();
```

Comment:
Hey Preston,
Could we name it isHeadVersion() instead?

The reason is that, as you say, "isHead" is the inverse of isDraft() but in reality we are going to use "isDraft" with a more complex purpose (for WorkflowModel), so it might be confusing to have the name of one being the inverse of the other if the reality is more complicated.